.github/dependabot.yml

```diff
@@ -88,3 +88,8 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
+
```

Comment:
Missing section for `/pallets/offchain-task-scheduler/runtime-api` although I admin there's nothing relevant in Cargo.toml there for the moment. Still we may add some dependencies which aren't coming from the workspace.